src/graphql/error/format_error.py

```diff
@@ -17,7 +17,11 @@ def format_error(error: "GraphQLError") -> Dict[str, Any]:
         raise TypeError("Received no error object.")
     formatted: Dict[str, Any] = dict(  # noqa: E701 (pycqa/flake8#394)
         message=error.message or "An unknown error occurred.",
-        locations=error.locations,
+        locations=(
+            [{"line": l.line, "column": l.column} for l in error.locations]
```

Comment:
Better use `loc` or `location` instead of the letter `l` - better readable and cannot be confused with the digit 1.

To be consistent in style with the outer dict, better write it like `dict(line=loc.line, column=loc.column)`, or rewrite the outer dict as literal expression.

We could also set `locations=[loc,_asdict() for loc in error.locations]`, but that will produce an OrderedDict, while we use ordinary dicts everywhere else (since they are ordered in Py 3.6+ anyway).